### DISCUSSION TITLE
Does KeePassXC support variant KDBX files?

### DISCUSSION THREAD

**Discussion by 1artist** [keepassxreboot/keepassxc#9953] [Q&A] [answered]
I share a common KDBX file containing 50 records between my three different desktop home PCs using Syncthing. Additionally, the same KDBX file is also shared to my Samsung cellphone using Syncthing. In short, I will feel a lot less paranoid if KeepassXC had the ability to automatically maintain a variant KDBX file specifically for use on my cellphone with the Financial folder records removed. Does KeepassXC have such a feature?  

I realize I could manually just create a copy of the desktop PC KDBX file and then manually delete from it all of my financial related records and use this copy only for my cellphone.  However, I still need KeepassXC to maintain a link to it so it can mirror any new records I add to my desktop KDBX file over to my cellphone KDBX file automatically. Can KeepassXC automatically maintain such a variant KDBX file for cellphone use only?  Any information welcome.

**Comment by droidmonkey** [comment, marked as answer]
If I were you, I would just have two databases. Sync both of them and only unlock your non-financial record one on the phone. Otherwise, you can use the KeeShare feature. Create a master db that has the financial records in it, then sync with the other database into a group. 

https://keepassxc.org/docs/KeePassXC_UserGuide#_database_sharing_with_keeshare
